packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -34,6 +35,18 @@ export async function createReactNativeConfigAsync({
     ...findProjectLocalDependencyRoots(projectConfig),
   };
 
+  // For Expo SDK 53 onwards, `react-native-edge-to-edge` is a transitive dependency of every expo project. Unless the user
+  // has also included it as a project dependency, we have to autolink it (transitive non-expo module dependencies are not autolinked).
+  const shouldAutolinkEdgeToEdge =
+    platform === 'android' && !Object.keys(dependencyRoots).includes('react-native-edge-to-edge');
```

Comment:
```suggestion
    platform === 'android' && !('react-native-edge-to-edge' in dependencyRoots)
```

"in" operater is faster enough to lookup a key from object